tests/foreman/api/test_subnet.py

```diff
@@ -32,7 +39,32 @@
 class ParameterizedSubnetTestCase(APITestCase):
     """Implements parametrized subnet tests in API"""
 
-    @stubbed()
+    @tier1
+    def test_positive_with_parameter(self):
```

Comment:
I guess `create` part is missing, it should be `test_positive_create_with_parameter`.